### PR TITLE
Package mkocaml.1.0

### PR DESCRIPTION
**Pull request by chrisnevers**
### `mkocaml.1.0`
Tool to generate OCaml projects

---
* Homepage: https://github.com/chrisnevers/mkocaml
* Source repo: git://github.com/chrisnevers/mkocaml.git
* Bug tracker: https://github.com/chrisnevers/mkocaml/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by hcarty** [comment]
Could you include a license so users know what they're allowed and not allowed to do with the project?

Thanks!

**Comment by chrisnevers** [comment]
@hcarty done! Thanks

File: packages/mkocaml/mkocaml.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+synopsis: "Tool to generate OCaml projects"
+maintainer: "Chris Nevers <christophernevers96@gmail.com>"
+authors: "Chris Nevers <christophernevers96@gmail.com>"
+homepage: "https://github.com/chrisnevers/mkocaml"
+bug-reports: "https://github.com/chrisnevers/mkocaml/issues"
+depends: [
+  "dune" {build}
```
Comment (mseri) [inline packages/mkocaml/mkocaml.1.0/opam]:
```suggestion
  "dune" {>= "1.11.0"}
```
According to your dune-project file you need at least dune 1.11.
Also dune cannot be tagged as build dependency: https://github.com/ocaml/opam-repository/issues/14185

File: packages/mkocaml/mkocaml.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "unix"
```
Comment (mseri) [inline packages/mkocaml/mkocaml.1.0/opam]:
```suggestion
  "base-unix"
```
This is part of the compiler itself. You can notify its use by depending on `base-unix`

**Comment by chrisnevers** [comment]
@mseri ah thanks!

**Comment by mseri** [comment]
Thanks. Feel free to announce the release on the discuss.ocaml.org forum. We have a community category and an announcement tag for the purpose

**Pull request merged**